modules/order/src/Adjustment.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce_price\Price;
+
+/**
+ * Represents an adjustment.
+ */
+final class Adjustment {
+
+  /**
+   * The adjustment type.
+   *
+   * @var string
+   */
+  protected $type;
+
+  /**
+   * The adjustment label.
+   *
+   * @var string
+   */
+  protected $label;
+
+  /**
+   * The price value object.
+   *
+   * @var \Drupal\commerce_price\Price
+   */
+  protected $price;
```

Comment:
if Price $amount is confusing, then that means we need to rename the Price object :)
Feels odd to have an adjustment price cause an adjustment is not something you buy.